File: day-5/solution.py
class Solution:
	def countSmaller(self, nums):
    	if len(nums) == 0:
        	return []
    	retu = [0]
    	num1 = [nums[-1]]
    	import bisect
    	for i in range(len(nums) - 2, -1, -1):
        	v = bisect.bisect_left(num1, nums[i])
        	retu.append(v)
        	num1 = num1[:v] + [nums[i]] + num1[v:]
    	return retu[::-1]	

File: day-5/solution.java
class Solution {
	public List<Integer> countSmaller(int[] nums) {
    	List<Integer> ans = new ArrayList<>();
    	if(nums.length == 0)
        	return ans;
   	 
    	for(int i = 0; i < nums.length - 1; i++){
        	int count = 0;
        	for (int j = i+1; j < nums.length; j++){
            	if(nums[i] > nums[j])
                	count++;
        	}
        	ans.add(count);
    	}
    	//Last number in nums always return 0
    	ans.add(0);
    	return ans;
	}
}
